smart-engine/src/main/java/org/smartdata/server/engine/CmdletManager.java

```diff
@@ -405,6 +408,29 @@ public long submitCmdlet(CmdletDescriptor cmdletDescriptor) throws IOException {
     return cmdletInfo.getCid();
   }
 
+  public void checkWhitelist(CmdletDescriptor cmdletDescriptor) {
```

Comment:
Can we move this method into WhitelistHelper? Thus, only call a method inside the class and let it return true/false to get the judgement result.